ua_ilab_tools/ilab_api.py

```diff
@@ -6,6 +6,7 @@
 
 class IlabApi(ua_generic_rest_api.GenericRestApi):
```

Comment:
I would suggest with this project that we add some unittests. We need to update the old ones, but I think unittests here are worth the work for them.